auth/credentials/internal/externalaccount/externalaccount.go

```diff
@@ -287,9 +288,9 @@ func (tp *tokenProvider) Token(ctx context.Context) (*auth.Token, error) {
 		SubjectToken:       subjectToken,
 		SubjectTokenType:   tp.opts.SubjectTokenType,
 	}
-	header := make(http.Header)
-	header.Set("Content-Type", "application/x-www-form-urlencoded")
-	header.Add("x-goog-api-client", getGoogHeaderValue(tp.opts, tp.stp))
+	hdr := make(http.Header)
+	hdr.Set("Content-Type", "application/x-www-form-urlencoded")
+	hdr.Add(header.GOOGLE_API_CLIENT_HEADER, getGoogHeaderValue(tp.opts, tp.stp))
```

Comment:
Do we need to send the new values here too?

Reply:
@arithmetic1728 Should this external account STS exchange be given the headers, and if so, which values? I couldn't find a similar flow in googleapis/google-auth-library-python. Please link if you know of one.

Reply:
I see that @aeitzman added metrics support for BYOID. Alex, is there some doc that details what headers to send?

https://github.com/googleapis/google-auth-library-python/pull/1332/files

Reply:
Ok looks like the code for sending x-goog-api-client headers for BYOID scenarios is already in there.
Sijun's design doesn't cover BYOID because Alex handled that separately. So no need to add the new values for BYOID files.